pkg/controller/clusterautoscaler/clusterautoscaler.go

```diff
@@ -96,6 +97,10 @@ func ScaleDownArgs(sd *v1alpha1.ScaleDownConfig) []string {
 		ScaleDownDelayAfterFailureArg.Value(sd.DelayAfterFailure),
 	}
 
+	if sd.UnneededTime != "" {
```

Comment:
Let's also add a unit test case that validates a config with/without this value.

Reply:
We should actually probably add `omitEmpty` to all of the scale down argument fields in the resource, or even make them pointers. Then fix the `Value()` method here to return an empty string if it gets `nil` or an empty string. Not sure if we do that here or in a follow up.

Reply:
I would vote in a follow-up.